rust-runtime/aws-smithy-http-server/Cargo.toml

```diff
@@ -32,7 +32,9 @@ regex = "1.0"
 serde_urlencoded = "0.7"
 thiserror = "1"
 tokio = { version = "1.0", features = ["full"] }
-tower = { version = "0.4" }
+tower = "0.4"
+tower-layer = "0.3"
```

Comment:
Can we just use the reexport of `tower-layer` from `tower`? https://docs.rs/tower/0.4.10/tower/#the-tower-ecosystem

Reply:
Yep, I can do that!